R/track_rust_source.R

```diff
@@ -0,0 +1,80 @@
+# See https://github.com/r-lib/pkgbuild/blob/2aaecf838eb023b788b7a31e267ac644d616dae0/R/compile-dll.R
```

Comment:
It is a reminder (mostly for myself). It points to the pkgbuild implementation, from which I drew inspiration. I did not copy any code though. pkgbuild is MIT.
This can be removed.